src/stats.js

```diff
@@ -14,43 +14,11 @@ const findFirstInTimeline = ({ timeline, fn }) => {
   return _.find(timeline, el => fn(el));
 };
 
-const self = (module.exports = {
-  getAveragePullRequestStats: async () => {
-    let previousPRs = await getPreviousPullRequests();
-
-    const statsResult = await Promise.all(
-      previousPRs
-        .filter(pr => pr.merged_at !== null)
-        .map(async pr => {
-          const stats = await self.getPullRequestStats(pr);
-          return stats;
-        })
-    );
-
-    const result = {}[
-      ('readyForReview', 'firstReview', 'firstApprovedReview', 'merged')
-    ].forEach(
-      column =>
-        (result[column] = {
-          fromPrCreation: averageValues(
-            statsResult.map(sr => sr[column].fromPrCreation)
-          ),
-          fromPreviousStep: averageValues(
-            statsResult.map(sr => sr[column].fromPreviousStep)
-          ),
-        })
-    );
-
-    return result;
-  },
-
-  getPullRequestStats: async pr => {
-    const prNumber = pr.number;
+module.exports = {
+  getStatsFromEventsTimeline: (pr, timeline) => {
```

Comment:
Nothing change really in this file , I mainly extracted the stats calculation to be able to add a simple test